docs/overhetalgoritmekader/index.md

```diff
@@ -1,83 +1,34 @@
-
-De motivatie om op deze ‘open source’ manier aan dit Algoritmekader te werken is enerzijds de omgeving van deze informatie te voorzien en anderzijds om de omgeving uit te nodigen om hier feedback op te geven. 
-Werken in Github is voor het team Algoritmekader nieuw en experimenteel. 
-Het biedt de mogelijkheid voor alle geïnteresseerden om hun kennis en zienswijze te delen. 
-Daarmee kan (nieuwe) kennis snel onderdeel worden gemaakt van het Algoritmekader. 
-Daardoor blijft het Algoritmekader zo actueel mogelijk en wordt de inhoud breed gedragen.
-Tegelijkertijd vraagt dit om een aangepaste werkwijze en is hier bepaalde expertise voor nodig. 
-Het begin is gemaakt en het team Algoritmekader is nog lerende om hier optimaal invulling aan te geven. 
-Voor nu betekent dit concreet dat het langer kan duren voordat wordt gereageerd op suggesties of toevoegingen. 
-Er wordt gewerkt aan het inrichten van goede processen om snel en kundig te reageren. 
+# Over het algoritmekader 
+Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin zoek je de regels en adviezen op per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een testversie. De definitieve versie verschijnt eind 2024.
+
+## Verantwoord gebruik van algoritmes en AI
+In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:
+- Wie is waarvoor verantwoordelijk?
+- Hoe voorkom je discriminatie (bias) in een algoritme?
+- Hoe verwerk je gegevens op een veilige manier?
+- Welke algoritmes controleer je op mensenrechtenschendingen?
+- Welke inkoopvoorwaarden spreek je af voor algoritmes van een externe partij?
+
+De informatie is bedoeld voor medewerkers van de rijksoverheid, provincies, gemeentes en waterschappen.
+
+## Informatie van de overheid
+Het Algoritmekader is een website van het Ministerie van Binnenlandse Zaken en Koninkrijksrelaties. Het team Data, AI en Algoritmes stelt de informatie samen, op basis van:
+- alle wetten en regels, zoals de [AI-verordening](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nieuwe-technologieen-data-en-ethiek/artificiele-intelligentie-ai/ai-verordening/)
+- belangrijke instrumenten, zoals de [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes) (Impact Assessment voor Mensenrechten bij de inzet van Algoritmes)
+- adviezen en ervaringen van experts, zoals wetenschappers
+
+## Iedereen mag meedenken
+Tot eind 2024 is het Algoritmekader een testversie. De informatie komt open source tot stand. Dat betekent dat iedereen zijn mening mag geven of een voorstel mag doen:
+- Geef je [vraag of wijziging](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/CONTRIBUTING/) door.
+- Sluit je aan bij een [werkgroep](https://algoritmes.pleio.nl/).
+- Doe mee aan een [online vergadering](https://algoritmes.pleio.nl/events) zoals onze sprintreview of demo.
```

Comment:
```suggestion
- Doe mee aan een [online vergadering](https://algoritmes.pleio.nl/events) zoals onze demo na een nieuwe release van het Algoritmekader.
```

Reply:
Is dat de enige vergadering waar mensen aan mee kunnen doen? Want dan is 'zoals' niet op zijn plaats.

Reply:
Er zijn er meerdere (niet allemaal online). We houden alleen nooit een sprintreview, maar wel een demo!